src/state/CycleInformationHooks.tsx

```diff
@@ -23,9 +21,9 @@ export function useDayOfCycle(): string {
     const start = new Date(startDate);
     const currentDate = new Date();
 
-    const diff = new Date(currentDate.getTime() - start.getTime());
+    const diff = currentDate.getDay() - start.getDay() + 1;
```

Comment:
`getDay' returns current day of week, not current day in Unix time stated from 1970

Reply:
Fixed.

Since I need a date (no time), I set the dates to the same time (00:00:00:00). Now when I look for the difference between dates the time is not taken into account.